includes/class-wc-youcanpay-webhook-handler.php

```diff
@@ -38,30 +38,148 @@ public function __construct() {
+
+        if (json_last_error() == JSON_ERROR_NONE) {
+            if (!array_key_exists('payload', $data)) {
+                return false;
+            }
+
+            if (!array_key_exists('transaction', $data['payload'])) {
+                return false;
+            }
+
+            if (!array_key_exists('payment_method', $data['payload'])) {
+                return false;
+            }
+
+            $transaction_id = null;
+            $transaction = new WC_YouCanPay_Transaction_Model($data['payload']['transaction']);
+            $payment_method = new WC_YouCanPay_Payment_Method_Model($data['payload']['payment_method']);
+            $payment_method_name = sprintf(__('YouCan Pay (%s)', 'youcan-pay'), $payment_method->get_name());
+
+            if (WC_YouCanPay_Payment_Method_Model::PAYMENT_METHOD_CASH_PLUS != $payment_method->get_name()) {
+                return false;
+            }
+
+            if (!isset($transaction)) {
+                WC_YouCanPay_Logger::info('arrived on process payment: transaction not exists', [
+                    'payment_method' => $payment_method_name,
+                    'code'           => '#0023',
+                    'transaction_id' => $transaction_id,
+                ]);
+
+                WC_YouCanPay_Webhook_State::set_last_webhook_failure_at(time());
+                WC_YouCanPay_Webhook_State::set_last_error_reason(__('The transaction does not exist', 'youcan-pay'));
+
+                return false;
+            }
+
+            $transaction_id = $transaction->get_id();
+            $order = wc_get_order($transaction->get_order_id());
+
+            if (!isset($order)) {
+                WC_YouCanPay_Logger::info('arrived on process payment: order not exists', [
+                    'payment_method' => $payment_method_name,
+                    'code'           => '#0024',
+                    'transaction_id' => $transaction_id,
+                    'order_id'       => $order->get_id(),
+                ]);
+
+                WC_YouCanPay_Webhook_State::set_last_webhook_failure_at(time());
+                WC_YouCanPay_Webhook_State::set_last_error_reason(
+                    __('The order received from the webhook does not exist', 'youcan-pay')
+                );
+
+                return false;
+            }
+
+            if ($transaction->get_status() === WC_YouCanPay_Transaction_Model::PAID_STATUS) {
+                WC_YouCanPay_Logger::info('payment successfully processed', [
+                    'payment_method' => $payment_method_name,
+                    'transaction_id' => $transaction->get_id(),
+                    'order_id'       => $order->get_id(),
+                    'order_total'    => $order->get_total(),
+                ]);
+
+                //TODO: Need to using WC_YouCanPay_Helper::set_payment_method_to_order() instead of $order->set_payment_method()
+
+                $order->set_payment_method($payment_method_name);
```

Comment:
Apply coding style by using camel case for naming this method

Reply:
PHP Coding Standards for WordPress

Use lowercase letters in variable, action/filter, and function names (never camelCase). Separate words via underscores. Don’t abbreviate variable names unnecessarily; let the code be unambiguous and self-documenting.